backend/dataall/core/permissions/__init__.py

```diff
@@ -1 +0,0 @@
-from dataall.core.permissions import api
```

Comment:
Why are we removing this import? I have not checked if it is explicitly necessary but i would assume it is

Reply:
Hm. It's in fact necessary for the other imports. So I better bring it back.